core/src/main/java/org/apache/carbondata/core/datamap/DataMapDistributable.java

```diff
@@ -34,7 +34,7 @@
 
   private String tablePath;
 
-  private Segment segment;
+  protected Segment segment;
```

Comment:
Already there is public `getSegment` method..so is this change required?..I think we can make use of `getSegment` to access wherever required

Reply:
reverted